www/dexpisvgNoaka.xslt

```diff
@@ -290,76 +291,15 @@
-                            <!-- First try to select the 'Value' attribute of the 'GenericAttribute'
-                            with the specific 'Name' -->
-                            <xsl:when
-                                test="$matchedElement/GenericAttributes/GenericAttribute[@Name='ObjectDisplayNameAssignmentClass']/@Value">
-                                <xsl:value-of
-                                    select="$matchedElement/GenericAttributes/GenericAttribute[@Name='ObjectDisplayNameAssignmentClass']/@Value" />
-                            </xsl:when>
-                            <!-- Selects the text for the offpageconnectors -->
-                            <xsl:when
-                                test="$matchedElement/*/GenericAttributes/GenericAttribute[@Name='ReferencedDrawingNumberAssignmentClass']/@Value">
-                                <xsl:value-of
-                                    select="$matchedElement/*/GenericAttributes/GenericAttribute[@Name='ReferencedDrawingNumberAssignmentClass']/@Value" />
-                            </xsl:when>
-                            <xsl:otherwise>
-                                <xsl:value-of
-                                    select="$matchedElement/GenericAttributes/GenericAttribute[@Name='ItemTagAssignmentClass']/@Value" />
-                            </xsl:otherwise>
-                        </xsl:choose>
-                    </xsl:variable>
-
-                    <xsl:variable name="IDValue" select="$matchedElement/@ID" />
-                    <xsl:variable name="attributeValue"
-                        select="GenericAttributes/GenericAttribute/@Value" />
-                    <xsl:variable name="docPath"
-                        select="concat('../../../../NOAKADEXPI/Symbols/Origo/', $attributeValue, '_Origo.svg')" />
-                    <xsl:variable name="label"
-                        select="GenericAttributes/GenericAttribute[@Name='ObjectDisplayNameAssignmentClass']/@Value" />
-                    <xsl:if
-                        test="not($docPath = '../../../../NOAKADEXPI/Symbols/Origo/BORDER_A1_Origo.svg')">
-                        <xsl:variable name="doc" select="document($docPath)" />
-						<xsl:apply-templates
-                            select="$doc//svg:g/*">
-                            <xsl:with-param name="labelParam" select="$displayNameValue" />
-                            <xsl:with-param name="idValue" select="$IDValue" />
-                        </xsl:apply-templates>
-                    </xsl:if>
-                    <xsl:apply-templates />
-                </symbol>
-            </xsl:for-each>
-        </defs>
     </xsl:template>
 
     <xsl:template match="svg:text[not(preceding::svg:text)]">
         <xsl:param name="labelParam" />
         <xsl:param name="idValue" />
         <xsl:if
-            test="string-length($labelParam > 0)">
+            test="string-length($labelParam) > 0 and not(contains($idValue, 'Nozzle'))">
```

Comment:
Er det riktig forstått at dette er en sjekk på streng-verdien på IDen til objektet? I tilfelle tror jeg det er tryggere å søke opp ID'en også sjekke om name til det er en Nozzle.

Reply:
Bruker nå ComponentClass, den ser ut til å være "Nozzle" for både Noaka og Aibel